packages/askar/src/wallet/AskarWallet.ts

```diff
@@ -131,6 +132,14 @@ export class AskarWallet implements Wallet {
     this.logger.debug(`Creating wallet '${walletConfig.id}`)
 
     const askarWalletConfig = await this.getAskarWalletConfig(walletConfig)
+
+    // Check if database exists
+    const { path: filePath } = uriFromWalletConfig(walletConfig, this.fileSystem.dataPath)
+    if (filePath && (await this.fileSystem.exists(filePath))) {
```

Comment:
Should we check whether the storage_type is postgres?

Reply:
For the moment it was not needed because we only return a path in sqlite, but I think this can be part of the improvements in Wallet API and configuration interfaces that we were discussing some weeks ago and probably we introduce in a near future (hopefully for 0.5.x).